public/js/test/utils/renderComponentFromFixture.js

```diff
@@ -7,7 +7,7 @@ const { fromJS, Map } = require("immutable");
 const { combineReducers } = require("redux");
 
 const fixtures = require("../fixtures");
-const configureStore = require("../../create-store");
+const configureStore = require("../../util/create-store");
```

Comment:
I don't mind it at the top-level because it's a core part of the initialization process, but it makes sense as a util too.